media-sound/picard/picard-2.1.3.recipe

```diff
@@ -4,35 +4,38 @@ HOMEPAGE="https://picard.musicbrainz.org/"
 COPYRIGHT="2004-2019 Robert Kaye, Lukas Lalinsky, Laurent Monin, \
 Sambhav Kothari, Philipp Wolfer and others"
 LICENSE="GNU GPL v2"
-REVISION="4"
+REVISION="5"
 SOURCE_URI="ftp://ftp.eu.metabrainz.org/pub/musicbrainz/picard/picard-$portVersion.tar.gz"
 CHECKSUM_SHA256="8e044fe68c44d345c19f98952c3c7f20c72da74dbac9ce7c7b0621e2d69885a7"
 SOURCE_DIR="picard-release-$portVersion"
 PATCHES="picard-$portVersion.patchset"
 ADDITIONAL_FILES="picard.rdef.in"
 
-ARCHITECTURES="x86 x86_64"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
-	picard = $portVersion
-	cmd:picard = $portVersion
+	picard$secondaryArchSuffix = $portVersion
+	app:Picard = $portVersion
 	"
 REQUIRES="
-	haiku
+	haiku$secondaryArchSuffix
+	lib:libdiscid$secondaryArchSuffix
 	cmd:python3.6
 	pyqt_python3
 	mutagen_python36
 	discid_python36
-	cmd:fpcalc
 	"
 
 BUILD_REQUIRES="
-	haiku_devel
-	cmd:gcc
-	cmd:msgfmt
-	cmd:python3.6
+	haiku${secondaryArchSuffix}_devel
 	setuptools_python36
-	pyqt_python3
+	"
+BUILD_PREREQUIRES="
+	cmd:fpcalc$secondaryArchSuffix
```

Comment:
@Begasus I don't think we need this for build, or was there a specific reason you added it?

Reply:
It was already in there (but as a left the comment on chromaprint, cmd:fpcalc isn't provided in the recipe, but the binary is in $prefix/bin so cmd:fpcalc is correct), for now I used this to bypass the problem, haven't checked if it's really needed

Reply:
@phw correct not needed so can be removed in the recipe